test/orbit_web/controllers/admin_controller_test.exs

```diff
@@ -0,0 +1,93 @@
+defmodule OrbitWeb.AdminControllerTest do
+  use OrbitWeb.ConnCase
+  import Ecto.Query
+  import Test.Support.Helpers
+
+  alias Orbit.BadgeSerial
+  alias Orbit.Employee
+  alias Orbit.Repo
+
+  @moduletag :authenticated
+  @moduletag groups: ["orbit-admin"]
+
+  describe "permissions" do
+    @tag groups: []
+    test "get fails if not admin", %{conn: conn} do
+      conn = get(conn, ~p"/admin/employee")
+      assert response(conn, :forbidden)
+    end
+
+    @tag groups: []
+    test "post fails if not admin", %{conn: conn} do
+      conn = post(conn, ~p"/admin/employee", %{})
+      assert response(conn, :forbidden)
+    end
+
+    test "post fails if not allow_test_data", %{conn: conn} do
+      reassign_env(:orbit, :allow_test_data?, false)
+      conn = post(conn, ~p"/admin/employee", %{})
+      assert response(conn, :method_not_allowed)
+    end
+  end
+
+  describe "employees" do
+    test "get loads form", %{conn: conn} do
+      conn = get(conn, ~p"/admin/employee")
+      assert response = html_response(conn, 200)
+      assert response =~ "<button"
```

Comment:
**nitpick:** doesn't really change the correctness of the test but the unbalanced brackets are bothering me 😅 
```suggestion
      assert response =~ "<button>"
```

Reply:
Oh this was intentional! It's just a string comparison, rather than like a CSS selector.  I don't want to assert on what HTML attributes the button will have, so this includes the `<`, which will definitely be right before the tag name, but not the `>`, which will probably be a few attributes later.

Apologies to your balanced bracket sensibilities.

Reply:
Ahhhh, I see. In that case I guess you could always match with a regular expression, otherwise I will just have to live with it I guess. 🙂